examples/cosmos/tendermint/TendermintModels.qnt

```diff
@@ -12,7 +12,7 @@ module TendermintModels {
   ) as n4_f1 from "./TendermintTest"
 
   import TendermintTest(
-    Corr = Set("p1", "p2", "p3"),
+    Corr = Set("p1", "p2"),
```

Comment:
Should there be some property/assert that says `Corr` and `Faulty` need to be disjoint?

Reply:
Ah, good, point. I have added an assumption